projects/js-packages/react-data-sync-client/src/DataSyncHooks.ts

```diff
@@ -29,12 +30,13 @@ export function invalidateQuery( key: string ) {
  * @see https://tanstack.com/query/v5/docs/react/reference/QueryClientProvider
  */
 export function DataSyncProvider( props: { children: React.ReactNode } ) {
-	return QueryClientProvider( {
-		client: queryClient,
-		...props,
-	} );
+	return React.createElement(
+		QueryClientProvider,
+		{ client: queryClient },
+		props.children,
+		React.createElement( ReactQueryDevtools, { initialIsOpen: true } )
```

Comment:
Noting this so we don't forget it.

During the call, we talked about making this closed by default.